dataproc/dataproc.go

```diff
@@ -0,0 +1,304 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command dataproc uses the Cloud Dataproc API to create a cluster,
+// submit a sample job, and delete the cluster when finished.
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+
+	dataproc "google.golang.org/api/dataproc/v1"
+	storage "cloud.google.com/go/storage"
+)
+
+const (
+	// Filename of example job (Python) file uploaded to Cloud Storage
+	exampleJobName = "test-file"
+
+	// Allows for full access to Google Cloud Platform products
+	scope = "https://www.googleapis.com/auth/cloud-platform"
+
+	// Compute URI base
+	computeUriBase = "https://www.googleapis.com/compute/v1/projects/%s/zones/%s"
```

Comment:
`s/Uri/URI`

Also, it's not really a base. It's a format string.

`computeURIFormat = ...`?